app/controllers/submissions/index.js

```diff
@@ -31,36 +35,88 @@ export default Controller.extend({
     },
   }),
 
-  piColumns: [
-    { propertyName: 'publication', title: 'Article', component: 'submissions-article-cell' },
-    { title: 'Award Number (Funder)', component: 'submissions-award-cell' },
-    { propertyName: 'user', title: 'Corr. Author', component: 'submissions-author-cell' },
-    { propertyName: 'repositories', title: 'Repo', component: 'submissions-repo-cell' },
-    { propertyName: 'userSubmittedDate', title: 'Last Update Date', component: 'date-cell' },
-    { propertyName: 'userSubmittedDate', title: 'Submitted Date', component: 'date-cell' },
+  piColumns: [{
```

Comment:
I clearly need to reconfigure my beautify plugin, sorry about these unnecessary changes.